internal/controller/infrastructure/kinkcluster_controller.go

```
@@ -25,6 +25,11 @@
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+const (
+	// name of control-plane finalizer
+	clusterFinalizer = "cluster.kink.anza-labs.com/finalizer"
```

golangci-lint (1.62.2)

const `clusterFinalizer` is unused

(unused)